packages/suite-data/files/translations/ru.json

```diff
@@ -116,6 +116,7 @@
   "RECEIVE_ADDRESS": "Адрес",
   "RECEIVE_ADDRESS_COINJOIN_DISALLOW": "Для создания дополнительных адресов для счета coinjoin вам необходимо убедиться, что вы уже получили биткоины на исходном адресе.",
   "RECEIVE_ADDRESS_FRESH": "Свежий адрес",
+  "RECEIVE_ADDRESS_LIMIT_REACHED": "",
```

Comment:
Empty string https://crowdin.com/translate/trezor-suite/103/en-ru?filter=basic&value=0#q=RECEIVE_ADDRESS_LIMIT_REACHED

Reply:
fixed in crowdin, will fix it in this PR too

Reply:
I tried it out and empty strings result to whatever we have in `messages.ts` so it's not a problem, but still good to clean it up.